app/Command/Docs/Html.hs

```diff
@@ -133,16 +130,16 @@ basicLayout title inner =
         toMarkup D.pursuitCssT
     H.body $ do
       H.div ! A.class_ "everything-except-footer" $ do
-        H.div ! A.class_ "top-banner clearfix" $ do
```

Comment:
I don't think I agree with hlint that redundant dos are a problem. In this case, I think I'd actually prefer to leave them in, as it means that you can go back and forth between elements with exactly one child vs more than one child without having to change the line which defines the parent. Can we keep ignoring the redundant do hint please?

Reply:
Dropped.